spec/command/setup_app_spec.rb

```diff
@@ -78,23 +78,4 @@
       expect(result[:stderr]).to match(/Binding identity to policy[.]+? done!/)
     end
   end
-
-  context "when using custom names for secrets" do
```

Comment:
The spec above ("binds identity to policy") now uses custom names, because with the global identifier we don't want to test using default names, as that will create a different secret/policy for each global identifier.